sportsdataverse/mlb/retrosplits.py

```
@@ -486,11 +480,12 @@ def retrosplits_player_pitching_by_platoon(first_season:int,last_season=None) ->
         season = i
         print(season)
         game_log_url = f"https://raw.githubusercontent.com/chadwickbureau/retrosplits/master/splits/pitching-platoon-{season}.csv"
-        #print(game_log_url)
-        resp = download(game_log_url)
-        resp_str = StringIO(str(resp, 'UTF-8'))
-        #season_game_log_df = pd.read_csv(resp_str,sep=",")
-        game_log_df = pd.read_csv(resp_str,sep=",")
+        try:
+            game_log_df = pd.read_csv(game_log_url,sep=",")
+        except:
+            game_log_df = pd.DataFrame()
+            print(f'Could not get pitching by platoon split stats for the {i} season. The file may not exist, or you may have an issue with your internet connection.')
```

see above re: swallowing errors